### PR TITLE
changed the comment lines to dart style

### PR DESCRIPTION
**Pull request by gozeloglu**
- I took https://github.com/flutter/gallery/issues/168 issue on me and changed the comment lines to make dart style.
- I re-wrote the comment lines as in [this document](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph) .

Closes https://github.com/flutter/gallery/issues/168

### CODE REVIEW

**Comment by gozeloglu** [comment]
> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Fflutter%2Fgallery%2Fpull%2F349) for more info**.

@googlebot I signed it!

**Comment by gozeloglu** [comment]
Hey @guidezpl . I changed the https://github.com/flutter/gallery/issues/168 and created PR but I forgot to add you as a reviewer. I am sorry for that. I just forgot it. Can you review my PR and give some feedback. If it is fine to merge master, can you merge it? If I need to add more, you can just tell me what should I do.

File: lib/layout/adaptive.dart [hunk 1]
```diff
@@ -5,11 +5,15 @@
...
 import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
 
+/// Returns a boolean by screen size.
+///
 /// Returns a boolean if we are on a medium or larger screen. Used to
 /// build adaptive and responsive layouts.
 bool isDisplayDesktop(BuildContext context) =>
     getWindowType(context) >= AdaptiveWindowType.m;
 
+/// Returns a boolean by window size
```
Comment (guidezpl) [inline lib/layout/adaptive.dart:15]: nit: needs a period at the end

File: lib/layout/adaptive.dart [hunk 2]
```diff
@@ -5,11 +5,15 @@
 import 'package:flutter/material.dart';
 import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
 
+/// Returns a boolean by screen size.
```
Comment (guidezpl) [inline lib/layout/adaptive.dart:8]: a better description might be "Whether the window type is medium or large"

File: lib/layout/adaptive.dart [hunk 3]
```diff
@@ -5,11 +5,15 @@
...
+///
 /// Returns a boolean if we are on a medium or larger screen. Used to
 /// build adaptive and responsive layouts.
 bool isDisplayDesktop(BuildContext context) =>
     getWindowType(context) >= AdaptiveWindowType.m;
 
+/// Returns a boolean by window size 
+///
 /// Returns true if the window size is medium size. Used to build adaptive and responsive layouts.
 bool isDisplaySmallDesktop(BuildContext context) {
```
Comment (guidezpl) [inline lib/layout/adaptive.dart:18]: a better description might be "Whether the window of considered medium"

**Comment by gozeloglu** [comment]
Hi @guidezpl I made changes according to your comments. Can you look at it? Thanks in advance.

File: lib/layout/adaptive.dart [hunk 4]
```diff
@@ -5,14 +5,14 @@
 import 'package:flutter/material.dart';
 import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
 
-/// Returns a boolean by screen size.
+/// Returns a boolean value whether the window is considered medium or large size.
 ///
 /// Returns a boolean if we are on a medium or larger screen. Used to
```
Comment (guidezpl) [inline lib/layout/adaptive.dart:10]: "The returns a boolean if we are..." comments are now redundant so they can be removed

Comment (gozeloglu) [reply lib/layout/adaptive.dart]: Yes, you are right. I resolved it

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

**Comment by guidezpl** [comment]
Thanks for the contribution!

**Pull request merged**